front/src/config/i18n/fr.json

```diff
@@ -1760,6 +1828,41 @@
       "stop": "Stop",
       "record": "Enregistrer"
     },
+    "energy-sensor": {
+      "shortCategoryName": "Capteur d'énergie",
+      "voltage": "Tension",
+      "current": "Intensité",
+      "binary": "Commutateur",
+      "power": "Puissance",
+      "energy": "Energie",
+      "index": "Index"
+    },
+    "volume-sensor": {
+      "shortCategoryName": "Capteur de volume",
+      "decimal": "Volume (décimale)",
+      "integer": "Volume (entier)"
+    },
+    "currency": {
+      "shortCategoryName": "Monnaie",
+      "cost": "Coût"
```

Comment:
"Coût" is weird, I tried to use this feature without looking at this file, and I couldn't find it. 

Maybe the category should be like "Bourse" ou "Devise", with inside "Monnaie" 

Looks like this now:

<img width="896" alt="Screenshot 2022-05-03 at 09 51 47" src="https://user-images.githubusercontent.com/7365207/166427124-804cafdd-92a7-4173-9387-7407a1024bc3.png">

Reply:
Also, it should be a decimal